gluecodium/src/main/resources/templates/jni/utils/JniCallbackErrorCheckingImplementation.mustache

```diff
@@ -0,0 +1,96 @@
+{{!!
```

Comment:
Please use functions and classes from JniReference.h, JniCppConversionUtils.h and JniCallJavaMethod.h
I'll write down couple of examples in this file

Reply:
Done.